### ISSUE TITLE
Make Plots Compatible with %matplotlib notebook

### ISSUE THREAD

**Issue by adnanhemani** [data-8/datascience#538]
Currently, the plot dimensions seem to have issues when using `%matplotlib notebook`. This magic function will allow us to make all plots interactive by default. However, the plot dimensions are causing problems out-of-the-box at this time. Example:

<img width="935" alt="Screen Shot 2022-06-05 at 6 15 39 PM" src="https://user-images.githubusercontent.com/10746770/172074662-6a823049-3caa-467e-8978-17dc9d91fa1a.png">

**Comment by SakshiSingh25** [comment]
Hi, I would like to work on this issue. Would you guide me in the further process? 
Thanks

**Comment by Vallabh100** [comment]
That's what _**%matplotlib notebook**_ command do right? We use this command to get zoom-able and resize-able interactive plots. So even though plot dimensions do cause problem initially, we can resize the plot. If you are facing timeout problem in notebook , just by using _**%matplotlib**_, it open separate prompt of plot and there u can visualize it. See below pic-
![image](https://github.com/data-8/datascience/assets/108627108/a015467a-448b-40ea-985e-4a125b004756)

**Comment by adnanhemani** [comment]
Yes, you are right that by we can resize and/or pop-out the plot - but we should fix this (if it is possible) to work with the majority of our plots right out of the box without the user needing to re-size or zoom on the plot
